dev.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:carbon as builder
+
+WORKDIR /usr/src/app/
```

Comment:
why such a dir? 👀

Reply:
I prefer /usr/src/ because this is the default directory for deploy application etc